torch/_refs/special/__init__.py

```diff
@@ -33,10 +35,10 @@
     type_promoting_args=("self",),
     type_promotion_kind=utils.ELEMENTWISE_TYPE_PROMOTION_KIND.INT_TO_FLOAT,
 )
-def logit(self: Tensor, eps: Optional[float] = None) -> Tensor:
+def logit(self: TensorLikeType, eps: Optional[float] = None) -> TensorLikeType:
     if eps is None:
         eps = -1.0
     lo = eps
     hi = 1 - eps
-    self = torch.clamp(self, lo, hi)
-    return (self / (1 - self)).log()
+    self = refs.clamp(self, lo, hi)
+    return refs.log(refs.true_divide(self, refs.sub(1, self)))
```

Comment:
@mruberry Given that the context manager exists now, we should *prefer* using the `torch` API calls as this ensures that the decomposition in question is using the limited API supported by torch and not the expanded API from refs. Is this just to work around the local problem that ref consistency tests don't work? I'd much rather we dupe the tests in that case.

Reply:
It's that the meta tests weren't working; I did duplicate the consistency tests